docs/docs/dart_api/manager.md

```diff
@@ -118,6 +118,23 @@ Any rows that meet the specified condition will be deleted.
 
 {{ load_snippet('manager_delete','lib/snippets/dart_api/manager.dart.excerpt.json') }}
 
+
+## Annotations
+
+It's quite common that you want to perform some complex calculations you data. Drift provides an easy way to write SQL expressions which can be read and/or used in filters and orderings.
+See the [expression](expressions.md) documentation for more information on how to write expressions.
+
+{{ load_snippet('manager_annotations','lib/snippets/dart_api/manager.dart.excerpt.json') }}
```

Comment:
One thing I've noticed in this snippet in particular, not sure if it's related to the snippet builder or this snippet. The comments have [a different indentation](https://deploy-preview-3234--moor.netlify.app/dart_api/manager/#annotations) on the site than the code. Is there an easy way to fix that?